bftengine/tests/simpleTest/scripts/simpleTest.py

```diff
@@ -0,0 +1,465 @@
+        cmd = [self.executable]
+        cmd.extend(self.params)
+
+        g_logger.debug("Run command: {}".format(cmd))
+
+        self.out_file = os.path.join(
+            self.log_dir, self.friendly_name + "_" + str(self.uid) + "_out.log")
+
+        FNULL = open(os.devnull, 'w')
+        with io.open(self.out_file, "wb", 8196) as out_writer, \
+        io.open(self.out_file, "br", 8196) as out_b_reader, \
+        io.open(self.out_file, "r", 8196) as out_t_reader:
+            self.process = subprocess.Popen(cmd,
+                                        bufsize=8196,
+                                        stdout=out_writer,
+                                        stderr=FNULL)
+            # output only client messages
+            if self.is_waitable:
+                while self.process.poll() is None:
+                    t = out_t_reader.readline()
+                    self.invoke_on_new_line(t)
+                    if not self.no_console:
+                        sys.stdout.buffer.write(out_b_reader.read())
+                #read leftovers
+                if not self.no_console:
+                    sys.stdout.buffer.write(out_b_reader.read())
+                lines = out_t_reader.readlines()
+                for line in lines:
+                    self.invoke_on_new_line(line)
+
+    def wait(self):
+        self.thread.join()
+
+    def stop(self):
+        self.process.terminate()
+        self.process.communicate()
+
+    def kill(self):
+        self.process.kill()
+        self.process.communicate()
+
+
+class TestConfig:
+    iterations_done = 0
+    leader_killed = False
+    iterations_counter_lock = threading.Lock()
+    config_run_timer = None
+    config_last_update_time = None
+
+    def __init__(self, num_rep, num_cl, faulty, slow, vc_enabled, vc_timeout,
+                 num_it, env, config_name, num_of_rep_to_launch, customize_fn):
+        self.num_of_replicas = num_rep
+        self.num_of_clients = num_cl
+        self.num_of_faulty = faulty
+        self.num_of_slow = slow
+        self.view_change_enable = vc_enabled
+        self.view_change_timeout = vc_timeout
+        self.num_of_iterations = num_it
+        self.env = env
+        self.waitables = []
+        self.non_waitables = []
+        self.process_outputs = []
+        self.config_name = config_name
+        self.num_of_replicas_to_launch = num_of_rep_to_launch
+        self.on_new_line_callback = self.on_new_line_vc if vc_enabled else \
+            self.on_new_line
+
+        self.config_timeout = 60000 + (0 if not vc_enabled else vc_timeout * 5)
+        self.customize_fn = customize_fn
+
+    @staticmethod
+    def reset_globals():
+        TestConfig.iterations_done = 0
+        TestConfig.leader_killed = False
+        TestConfig.config_run_timer = None
+        TestConfig.config_last_update_time = None
+
+    def prepare(self):
+        exec_dir = os.path.abspath(self.env.binary_dir)
+        time_stamp = time.strftime("%d_%m_%Y_%H_%M_%S")
```

Comment:
Please change this to an ISO8601-like format, so that the directories sort lexicographically. e.g. `%Y%m%dT%H%M%S`